english/33hours/explore-outside/outside.md

```diff
@@ -5,4 +5,5 @@ left and your right. You stand there several minutes, but see no cars at all.
 The motel parking lot is completely empty. It looks to be abandoned. There are no other buildings around for as far as
 you can see.
 
-[Go back inside and explore the motel room.](../explore-room/room.md)
\ No newline at end of file
+[Go back inside and explore the motel room.](../explore-room/room.md)
+[Start walking east, on the highway](./explore-east.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.